.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,81 @@
+name: End 2 End Tests (Spaceflights Quickstart)
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Setup python
+      uses: actions/setup-python@v2.2.1
+      with:
+        python-version: 3.8
+
+    - name: Setup virtualenv
+      run: |
+        python -V
+        python -m pip install virtualenv
+        virtualenv venv
+        source venv/bin/activate
+   
+    - name: Initialize kedro spaceflights project
+      run: |
+        pip install . 'kedro<0.18'
+        kedro new --starter spaceflights --config tests/e2e/starter-config.yml --verbose --checkout=0.17.6
+
+    - name: Install project dependencies
+      run: |
+        cd ./spaceflights
+        echo "git+https://github.com/getindata/kedro-vertexai.git@$GITHUB_SHA" >> src/requirements.txt
+        echo "kedro-docker" >> src/requirements.txt
+        sed -i '/kedro-telemetry/d' src/requirements.txt
+        cat src/requirements.txt
```

Comment:
Leftover?

Reply:
You mean this `cat`? I left it intentionally for debugging purposes